models/enterprise_sonic/ntp/sonic_ntp.yaml

```diff
@@ -44,8 +44,10 @@ DOCUMENTATION: |
           servers:
             type: list
             elements: dict
+            required_together: [['minpoll', 'maxpoll']]
             description:
               - List of NTP servers.
+              - minpoll and maxpoll are required to configure together.
```

Comment:
Grammar change; Please change this line to:

```
              - minpoll and maxpoll are required to be configured together.
```

Reply:
Changed. Thanks!